### PR TITLE
Add Paxe (BEP20)

### PR DESCRIPTION
**Pull request by rez404**
Adding info for BEP20 token 'Paxe'.

PR created by <a href="https://assets.trustwallet.com">TW Assets Management</a>

### CODE REVIEW

**Comment by rez404** [comment]
Our Audits : 
https://skynet.certik.com/projects/paxe
https://www.cyberscope.io/audits/paxe

**Comment by PaxeFi** [comment]
Liquidity Farming DApp: https://app.sakaivault.io/farming

**Comment by rez404** [comment]
Tokenomics: https://docs.paxe.io/introduction/tokenomics

**Comment by rez404** [comment]
We listed on MEXC
https://www.mexc.com/exchange/PAXE_USDT

**Comment by rez404** [comment]
@Cryptocool1 everything ok?

**Comment by Cryptocool1** [comment]
• What is the purpose of this token? How will it benefit Trust Wallet users?
 • How is the token being distributed? Are you conducting an airdrop?
 • Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?

**Comment by rez404** [comment]
• What is the purpose of this token? How will it benefit Trust Wallet users?
PAXE helps Trust Wallet users easily stake and farm tokens to maximize their earnings, all from one simple platform. It’s a convenient way to boost your returns.

• How is the token being distributed? Are you conducting an airdrop?
Not have any airdrop. Here our tokenomics : https://docs.paxe.io/introduction/tokenomics#tokenomics

• Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
Yes we did presale on pinksale. Presale link: 
https://www.pinksale.finance/launchpad/bsc/0xAD58e19ba75EBb428CCdB588350f73fFef39DEAA

@Cryptocool1 thank you for your interest. If there are more questions, we are here. You can ask without hesitation.

**Comment by rez404** [comment]
@Cryptocool1 thank you for your interest. If there are more questions, we are here. You can ask without hesitation.

**Comment by rez404** [comment]
@Cryptocool1 whats the next? Everything ok?

**Comment by rez404** [comment]
@Cryptocool1 whats the next? Everything ok? We get an error can you check please?

File: blockchains/smartchain/assets/0xd2A3eec06719D5Ac66248003B5488E02165dd2fa/info.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "decimals": 18,
+    "website": "https://www.paxe.io/",
+    "description": "PAXE Protocol is a cutting-edge native yield platform that provides access to cost-effective and secure high-quality yield opportunities across the industry.",
+    "explorer": "https://bscscan.com/token/0xd2A3eec06719D5Ac66248003B5488E02165dd2fa",
+    "status": "active",
+    "id": "0xd2A3eec06719D5Ac66248003B5488E02165dd2fa",
+    "links": [
+        {
+            "name": "twitter",
+            "url": "https://x.com/Paxe_fi"
```
Comment (rez404) [inline blockchains/smartchain/assets/0xd2A3eec06719D5Ac66248003B5488E02165dd2fa/info.json:14]: https://twitter.com/paxe_fi

**Review by rez404 — COMMENTED** [review]
Twitter link

**Comment by rez404** [comment]
@cryptomanz1 we get an error. fixed now can you check please?

**Comment by rez404** [comment]
@Cryptocool1  @cryptomanz1  we got an error. fixed now, can you check please?

**Comment by rez404** [comment]
@Cryptocool1 any progress?

**Comment by rez404** [comment]
Hello again,

Could you check and help to finalize that process please ?

If you need anything from our side, please do not hesitate to request.

@Cryptocool1 
@cryptomanz1 

Thanks for understanding and your care.

Best regards

**Comment by rez404** [comment]
https://coinmarketcap.com/currencies/paxe/

**Pull request merged**